7_react/react1/week2/homework/todo-list/src/App.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import TodoList from "./components/todolist";
+import { useState } from "react";
+import { useEffect } from "react";
+
+const initialTodos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017",
+    done: false,
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017",
+    done: false,
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017",
+    done: false,
+  },
+];
+
+function App() {
+  const [timer, setTimer] = useState(0);
+  const [todos, setTodos] = useState(initialTodos);
+
+  const updateTimer = () => {
+    setTimer((prevTime) => prevTime + 1);
+  };
+  useEffect(() => {
+    const intervalId = setInterval(updateTimer, 1000);
+    return () => clearInterval(intervalId);
+  }, []);
+
+  const addTodo = () => {
+    const randomTodo = {
+      id: Date.now(),
+      description: `Random Task`,
+      done: false,
```

Comment:
Also try to add the deadline property with a default value in here(since you have it at the start).